src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/request.html.twig

```diff
@@ -245,6 +245,22 @@
             </div>
         </div>
 
+        <div class="tab {{ profile.collector('dotenv').envs is empty ? 'disabled' }}">
```

Comment:
this would break in case the dotenv panel is not registered

Reply:
@stof, you're right! I've added an `if` expression to check whether the `dotenv` collector is registered.